### PR TITLE
add a token refresh integration test in messaging

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

File: Example/Messaging/IntegrationTests/FIRMessagingTokenRefreshTests.swift [hunk 1]
```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (paulb777) [inline Example/Messaging/IntegrationTests/FIRMessagingTokenRefreshTests.swift:2]: 2020

File: FirebaseMessaging.podspec [hunk 2]
```diff
@@ -63,4 +63,13 @@ device, and it is completely free.
     }
     unit_tests.dependency 'OCMock'
   end
+
+  s.test_spec 'integration' do |int_tests|
+    int_tests.platforms = {:ios => '8.0', :osx => '10.11', :tvos => '10.0'}
+    int_tests.source_files = 'Example/Messaging/IntegrationTests/*.swift'
+    int_tests.requires_app_host = true
+    int_tests.resources = 'Example/Messaging/Tests/IntegrationTests/Resources/1mb.dat',
```
Comment (paulb777) [inline FirebaseMessaging.podspec:71]: 1mb.dat not needed for messaging

File: Example/Messaging/IntegrationTests/FIRMessagingTokenRefreshTests.swift (hunk 1 above)
Comment (chliangGoogle) [reply Example/Messaging/IntegrationTests/FIRMessagingTokenRefreshTests.swift:2]: Done.

File: FirebaseMessaging.podspec (hunk 2 above)
Comment (chliangGoogle) [reply FirebaseMessaging.podspec]: Done.

**Comment by chliangGoogle** [comment]
@paulb777 @wilhuff Not sure why FireStore check failed, even tho I didn't touch any Firestore code.

**Comment by paulb777** [comment]
@chliangGoogle Firestore's GHA is triggered by a change in `build.sh` and there is a known issue in master in the asan tests.  You can safely disregard.

**Pull request merged**